### DISCUSSION TITLE
What is this 5 second play limit?

### DISCUSSION THREAD

**Discussion by Koozu1** [jagrosh/MusicBot#1378] [Questions] [answered]
I try to play a song and get response "💡 This track (Song) is longer than the allowed maximum: 02:28 > 00:05"

**Comment by jagrosh** [comment, marked as answer]
You set the `maxtime` value in your [config file](https://jmusicbot.com/config/).

**Reply by Koozu1** [reply]
Ah thank you sir! Thought that was for maximum time of the bot staying on channel when no users are present, or similar :D
